cmd/root.go

```diff
@@ -13,13 +15,17 @@ import (
 )
 
 const (
-	flagVerbose = "verbose"
-	flagNoColor = "no-color"
+	flagVerbose  = "verbose"
+	flagNoColor  = "no-color"
+	flagDetached = "detached"
 )
 
+var Interactive bool = true
```

Comment:
```suggestion
var interactive bool = true
```
can scope it to this pkg only

Reply:
It's needed in `main` as well.